core/domain/rights_manager_test.py

```diff
@@ -443,6 +443,42 @@ def test_changing_viewability_of_exploration(self):
         self.assertFalse(rights_manager.check_can_access_activity(
             self.user_b, exp_rights))
 
+    def test_reassign_higher_role_to_exploartion(self):
```

Comment:
Typo in ``exploration``. Ditto below.

Please fix.

Reply:
Done.